deploy/cw-bundle.yaml

```diff
@@ -1926,6 +1932,8 @@ spec:
                           properties:
                             bucket:
                               type: string
+                            caBundle:
```

Comment:
I am not sure that we need to have it. We set certificates via secret only. @egegunes  WDYT?